jena-arq/src/main/java/org/apache/jena/sparql/core/mem/QuadTableForm.java

```diff
@@ -124,31 +96,21 @@ public PMapQuadTable get() {
 			return new PMapQuadTable(name()) {
 
 				@Override
-				protected Quad quad(final Node s, final Node p, final Node o, final Node g) {
-					return Quad.create(g, s, p, o);
-				}
+                public Quad tuple(final Quad q) {
+                    return create(q.getSubject(), q.getPredicate(), q.getObject(), q.getGraph());
```

Comment:
These criticisms are sadly accurate. I seem to have improved in one way by creating other defects. I'm going to close this PR and see if I can solve the problems I identified (repeated code, unclear responsibility) in a better, tighter way, perhaps involving `ColumnMap`.